src/test/java/com/rackspace/ceres/app/web/MetadataControllerTest.java

```diff
@@ -78,8 +78,9 @@ public void testGetTagValues() {
         .thenReturn(Mono.just(List.of("linux")));
 
     webTestClient.get().uri(
-        uriBuilder -> uriBuilder.path("/api/metadata/tagValues").queryParam("tenant", "t-1")
+        uriBuilder -> uriBuilder.path("/api/metadata/tagValues")
             .queryParam("metricName", "metric-1").queryParam("tagKey", "os").build())
+        .header("X-Tenant-Id", "t-1")
```

Comment:
Please duplicate the test case and have one still use the tenant query parameter.